easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -199,8 +202,9 @@ def get_toolchain_hierarchy(parent_toolchain):
     optional_toolchains = set(tc_class.NAME for tc_class in all_tc_classes if getattr(tc_class, 'OPTIONAL', False))
     composite_toolchains = set(tc_class.NAME for tc_class in all_tc_classes if len(tc_class.__bases__) > 1)
 
-    # the parent toolchain is at the top of the hierarchy
-    toolchain_hierarchy = [parent_toolchain]
+    # the parent toolchain is at the top of the hierarchy, we need a copy so that adding capabilities (below) doesn't
```

Comment:
comment nitpicking (yes, it's a thing): start the `we need a copy` on the line below